app/models/robot_world.rb

```diff
@@ -9,60 +8,49 @@ def initialize(database)
   end
 
   def create(robot)
-    database.transaction do
-      database['robots'] ||= []
-      # database['total'] ||= 0
-      # database['total'] += 1
-      # database total from number of keys in database['robots']
-      database['robots'] << {
-        "name" => robot[:name],
-        "city" => robot[:city],
-        "state" => robot[:state],
-        "avatar" => robot[:avatar],
-        "birthdate" => robot[:birthdate],
-        "date hired" => robot[:date_hired],
-        "department" => robot[:department],
-      }
-    end
-  end
-
-  def create_avatar(name)
-    '<img src="https://robohash.org/<%= @robot.name %>">'
+    database.execute("INSERT INTO robots (name, city, state, birthdate, date_hired, department)
+                      VALUES (?, ?, ?, ?, ?, ?);",
+                      robot[:name], robot[:city], robot[:state],
+                      robot[:birthdate], robot[:date_hired], robot[:department])
   end
 
   def all_robots
-    database.transaction do
-      database['robots'] || []
-    end
+    database.execute("SELECT * FROM robots;")
   end
 
   def all
     all_robots.map { |data| Robot.new(data) }
   end
 
-  def find_robot_named(name)
-  all_robots.find { |robot| robot["name"] == name }
+  def find_robot_named(id)
```

Comment:
this method name doesn't match the purpose or the behavior of this method. perhaps `find_robot` might be fine